examples/marketdata/marketdata.go

```diff
@@ -209,6 +236,9 @@ func main() {
 		{Name: "news", Func: news},
 		{Name: "auctions", Func: auctions},
 		{Name: "crypto_quote", Func: cryptoQuote},
+		{Name: "crypto_perp_quote", Func: cryptoPerpsQuote},
+		{Name: "crypto_perp_trade", Func: cryptoPerpsTrade},
+		{Name: "crypto_perp_bar", Func: cryptoPerpsBar},
```

Comment:
[minor] Could you please right-pad in line 248 now that we have more than 9 entries here?
``` go
fmt.Printf("[ %2d ] %s\n", i, e.Name)
```

Reply:
Done.